htdocs/contact/list.php

```diff
@@ -76,7 +76,7 @@
 if (!empty($conf->socialnetworks->enabled)) {
 	foreach ($socialnetworks as $key => $value) {
 		if ($value['active']) {
-			$search_{$key} = GETPOST("search_".$key, 'alpha');
+			$search_[$key] = GETPOST("search_".$key, 'alpha');
```

Comment:
curly bracket are used for arrays so an array called 'search_'
Here we want to edit the variable call $search_abc where abc is value of $key.

Reply:
Ok, I seen now.
I'll try to be more careful next time.